src/main/java/au/org/ala/biocache/dao/SearchDAOImpl.java

```diff
@@ -985,7 +985,13 @@ private void addPostProcessingFields(DownloadRequestParams downloadParams, Downl
                 } catch (Exception e) {
                     logger.error("error getting assertions facet for download: " + downloadParams, e);
                 }
+            } else {
```

Comment:
It is not clear in this function:
- `qa=none` excludes all assertions
- `qa=all` includes all assertions that are present for the query
- `qa=includeall` includes all available assertions

After checking api.ala.org.au it does appear that this `else` block is valid, although I don't think it used to work.

Please add an exclusion for `qa=none`.

Reply:
An exclusion for `qa=none` on the `else` block is redundant as it is already within `if (!"none".equals(downloadParams.getQa()))` block. 

The code from current master does include the assertion names in the `qa` param. https://github.com/AtlasOfLivingAustralia/biocache-service/blob/cf41784f1f57e39f7af7232ebd360ee966c34f6d/src/main/java/au/org/ala/biocache/dao/SearchDAOImpl.java#L1078-L1081